MANIFEST.in

```diff
@@ -19,8 +17,6 @@ recursive-include docs *
 recursive-include examples *
 recursive-include licenses *
 recursive-include cextern *
-recursive-include scripts *
-recursive-include static *
```

Comment:
We shouldn't need any of the includes in this file at all, because setuptools-scm handles all of it.

Reply:
Are you sure? Since when?

Reply:
since forever, we could have removed these when we did the APE 17 migration.